src/main/java/com/clairvoyant/weather/document/WeatherDetails.java

```diff
@@ -0,0 +1,25 @@
+package com.clairvoyant.weather.document;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+/**
+ * @author Gufran Khan
+ * @version 1.0
+ * @date 17-08-2021 16:20
+ */
+@Document
+@Data
```

Comment:
Please don't use `@Data`. Instead, use the specific annotations that are required like `@Getter` and`@Setter`